airflow/providers/google/cloud/hooks/vertex_ai/custom_job.py

```diff
@@ -2178,3 +3300,239 @@ def list_custom_jobs(
             metadata=metadata,
         )
         return result
+
+    @GoogleBaseHook.fallback_to_default_project_id
+    @deprecated(
+        reason="Please use `PipelineJobHook.delete_pipeline_job`",
+        category=AirflowProviderDeprecationWarning,
+    )
+    def delete_pipeline_job(
+        self,
+        project_id: str,
+        region: str,
+        pipeline_job: str,
+        retry: Retry | _MethodDefault = DEFAULT,
+        timeout: float | None = None,
+        metadata: Sequence[tuple[str, str]] = (),
+    ) -> Operation:
+        """
+        Delete a PipelineJob.
+
+        This method is deprecated, please use `PipelineJobHook.delete_pipeline_job` method.
+
+        :param project_id: Required. The ID of the Google Cloud project that the service belongs to.
+        :param region: Required. The ID of the Google Cloud region that the service belongs to.
+        :param pipeline_job: Required. The name of the PipelineJob resource to be deleted.
+        :param retry: Designation of what errors, if any, should be retried.
+        :param timeout: The timeout for this request.
+        :param metadata: Strings which should be sent along with the request as metadata.
+        """
+        client = self.get_pipeline_service_client(region)
+        name = client.pipeline_job_path(project_id, region, pipeline_job)
+
+        result = client.delete_pipeline_job(
+            request={
+                "name": name,
+            },
```

Comment:
comma was not added elsewhere. let's stick with one convention
```suggestion
            request={"name": name},
```

Reply:
These are pre-commit changes, not mine.